### DISCUSSION TITLE
Not able to install Salesforce extension properly

### DISCUSSION THREAD

**Discussion by sfsagar** [microsoft/vscode-cpptools#12025] [General]
Hi All, 

facing issue while installing Salesforce extension pack ,shing the error as shown in the image.
![image](https://github.com/microsoft/vscode-cpptools/assets/97340699/a518b95b-0709-4021-b815-5d8d3b6fa900)

Just completely deleted VS code, salesforce CLI and JDK. Now again downloaded and installed but still this extension pack issue is available due to whiich i could not even connect to my org in default org option.

Please anyone can suggest me how can i resolve the issue.

**Comment by bobbrow** [comment]
Sorry @sfsagar, this repo is for the C++ extension.  The repo you're looking for might be this one: https://github.com/forcedotcom/salesforcedx-vscode
